src/localStorage.js

```diff
@@ -7,6 +7,15 @@ export const saveLogin = state => {
   }
 };
 
+export const saveTeams = state => {
+  try {
+    const serializedstate = JSON.stringify({ teams: state });
+    localStorage.setItem("teams", serializedstate);
+  } catch (err) {
+    //Nothing should happen.
```

Comment:
What happens when a team couldn't be saved? Don't you return feedback for the user?